pages/blog/google-season-of-docs-2022.md

```diff
@@ -0,0 +1,95 @@
+
+
+### ⌛ Timeline
+The project itself will take approximately 3-6 months to complete, depending on how many technical writers (TW) get involved. If we can get even just 2-4 TWs, then we can work on both migrating the CLI and Tools documentation. But if we can only get 1-2 folks, it would be hard to do more than the migration of CLI or Tools docs. Ideally, the timeline could look like the following:
+
+- **May:** Orientation on how to contribute to AsyncAPI Inititiave, how Docs issues are organized, detail how we're migrating our CLI and Tools Docs, and assign good `first-time-tickets` to get each new TW contributor started. 
+- **June - August:**	Each TW goes through designated issues marked for both first time contributors and work set aside for `GSoD 2022`. Each TW starts creating documentation for their individual issues assigned/selected. 
+- **September - October:** We determine if we're going to be able to complete both CLI and Tools Docs, depending on how many TWs are in our group and how much they've been able to complete so far. We re-align priorities as needed and asses what is missing to reach our 2022 IA change goals for AsyncAPI Docs.
+- **November:**	Project completion and all contributors receive some swag! 
+
+
+### 💸 Project budget
+We have set aside 2 mentors for now, for our 1 project on improving our IA. For this we would like to request a US $5000 budget to Google, should we be selected. 
+
+___ 
+
+## 👉🏽 Get started contributing to AsyncAPI Docs Today
+Last but not least, don't forget that code isn't the only way to contribute to OSS; Dev Docs are a **huge** help that benefit the entire OSS ecosystem. At AsyncAPI, we value Doc contributions as much as every other type of contribution. ❤️
+
+To get started as a Docs contributor:
+1. Familiarize yourself with our [project's Contribution Guide](https://github.com/asyncapi/community/blob/master/CONTRIBUTING.md) and our [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).
+2. Head over to our Docs GH Board [here](https://github.com/orgs/asyncapi/projects/8).
+3. Pick an issue you would like to contribute to and leave a comment introducing yourself. This is also the perfect place to leave any questions you may have on how to get started. 
+4. If there is no work done in that Docs issue yet, feel free to open a PR and get started!
+
+### 🏷 Tag me in your AsyncAPI Doc PRs
+Do you have a documentation contributor question and you're wondering how to tag me into a GitHub discussion or PR? Never fear!
+
+Tag me in your AsyncAPI Doc PRs or [GitHub Discussions](https://github.com/asyncapi/community/discussions/categories/docs) via my GitHub handle, [`/alequetzalli`](https://github.com/alequetzalli) 🐙.
```

Comment:
```suggestion
Tag me in your AsyncAPI Doc PRs or [GitHub Discussions](https://github.com/asyncapi/community/discussions/categories/docs) via my GitHub handle, [alequetzalli](https://github.com/alequetzalli) 🐙.
```

I would not use code formatting for your username's link.

Reply:
@fmvilas   The `/` I used to mention my GH profile is actually copy/pasted [exactly how I did it in my last blog post with us](https://www.asyncapi.com/blog/changes-coming-docs#%F0%9F%8F%B7-tag-me-in-your-asyncapi-doc-prs), and there were no issues there. Ergo, we know that is not a problem :)

Since this was no problem in the previous post, let me know if you're really that sure you want it out. I personally think it's useful. 

<img width="500" alt="Screen Shot 2022-03-23 at 6 54 04 PM" src="https://user-images.githubusercontent.com/19964402/159826325-32fe5da5-67a8-4c09-80a5-a92193dd399a.png">